index.md

```diff
@@ -8,20 +8,10 @@ permalink: /
 
 ![Oculus Grandma](/img/OculusGrandma.jpg){:class="img-responsive" style="display:block; margin:auto"}
 
-
-## Overview ##
-Virtual reality (VR) technology transports us to real or synthetic places that may be inaccessible, breathtaking, complex beyond our wildest imagination, or just simple and relaxing. Applications include entertainment, social interaction, virtual travel, remote training, architectural walkthroughs, cultural appreciation, and learning enhancement.
-
-Although VR has been around for decades, the cost of entry has previously been high because of advanced, expensive equipment and computing resources. Thanks to widespread progress in display, sensing, and computational technology, the newest VR systems are cheap, lightweight, and easy to program. This has caused a flood of excitement as almost anyone can pick up a VR headset and start developing experiences.
-
 ## Course Objectives ##
-The purpose of this course is to provide students with both a deep understanding of the fundamentals of VR and to gain practical experience. Because VR tricks our brains by presenting synthetic stimuli to our senses, it is extremely challenging to develop and analyze VR systems that are both effective and comfortable. To get a handle on these issues, this course will fuse together knowledge from a variety of relevant topics, including computer graphics, tracking systems, and perceptual psychology. Some basic questions that motivate the course topics:
+Virtual reality (VR) technology transports us to real or synthetic places that may be inaccessible, breathtaking, complex beyond our wildest imagination, or just simple and relaxing. Inaccessible in prior decades due to expensive equipment and computing needs, VR systems are now cheap, easy to program, and applicable across almost all fields due to advances in display, sensing, and computational technology. Almost anyone can pick up a VR headset and start developing experiences!
 
-- How does one build a good VR experience?
-- How do VR systems work using current technology?
-- What is wrong with current systems?
-- How does the human body respond to VR?
-- What fundamentals could help you to shape the future?
+The purpose of this course is to provide students with both a deep understanding of the fundamentals of VR and to gain practical experience. Because VR tricks our brains by presenting synthetic stimuli to our senses, it is extremely challenging to develop and analyze VR systems that are both effective and comfortable. To get a handle on these issues, this course will fuse together knowledge from a variety of relevant topics, including computer graphics, tracking systems, and perceptual psychology. 
```

Comment:
Experience with what?
```suggestion
The purpose of this course is to provide students with both a deep understanding of the fundamentals of VR and to gain practical experience developing VR content. Because VR tricks our brains by presenting synthetic stimuli to our senses, it is extremely challenging to develop and analyze VR systems that are both effective and comfortable. To get a handle on these issues, this course will fuse together knowledge from a variety of relevant topics, including computer graphics, tracking systems, and perceptual psychology. 
```